externals/kyuubi-spark-monitor/src/main/scala/org/apache/kyuubi/engine/spark/monitor/KyuubiStatementMonitor.scala

```diff
@@ -36,17 +42,34 @@ object KyuubiStatementMonitor extends Logging{
    *      b. this queue's current size
    */
   // TODO: Capacity should make configurable
-  private val kyuubiStatementQueue = new ArrayBlockingQueue[KyuubiStatementInfo](10)
+  private val kyuubiStatementQueue = new ArrayBlockingQueue[KyuubiStatementInfo](maxCapacity)
+
+  /**
+   * This map store the relationship between jobId and jobInfo.
+   * When the job has finished, all we can get is jobId from the object-jobEnd.
+   * So we need to maintain a mapping relationship to store endTime and jobResult
+   * when this job has finished.
+   *
+   * Key is jobId, value is KyuubiJobInfo.
+   *
+   * Notice:
+   *    1. There have two kinds of threshold to trigger when to remove and dump items from this map:
+   *      a. time
+   *      b. this map's current size
+   */
+  // TODO: Capacity should make configurable
```

Comment:
It's better to create a new issue to track this `TODO`, and put that issue link here.

Reply:
It's a good idea. I will do it soon.